### PR TITLE
Improvements to the profile type in chat feature and bug fix for heads in tablist

### PR DESCRIPTION
**Pull request by CattoFace**
Renamed feature to "Player Symbols" as it also displays the players nether faction.
Improved symbol finding by checking the tab list as well.
Added symbol to guild chat
As part of checking the tab list, fixed player name stripping issue that prevented heads from being displayed.
#621 also tackles the player head bug but it was not merged and this new solution is somewhat cleaner.

### CODE REVIEW

**Review by ILikePlayingGames — APPROVED** [review]

**Comment by ILikePlayingGames** [comment]
@ImRodry this changes a string key and a string. Is that ok?

**Comment by CattoFace** [comment]
This feature is to add symbols to **chat**.
the symbols are shown by default on players head and in the tab list but not in chat.

**Comment by ImRodry** [comment]
Then call it “Show”, not “Add”

**Comment by CattoFace** [comment]
changed the string and merged with the recent pr

File: src/main/resources/lang/en_US.json [hunk 1]
```diff
@@ -220,7 +220,7 @@
     "showFeatureNamesOnHover": "Show Feature Names on Hover",
     "resetSalvagedEssencesAfterLeavingMenu": "Reset Salvaged Essence Counter After Closing Menu",
     "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys",
-    "showProfileTypeInChat": "Show Profile Type In Chat",
+    "showPlayerSymbolsInChat": "Show Player Symbols In Chat Messages",
```
Comment (ImRodry) [inline src/main/resources/lang/en_US.json:223]:
```suggestion
    "showPlayerSymbolsInChat": "Show Player Symbols In Chat",
```

**Review by ImRodry — APPROVED** [review]

**Pull request merged**